src/NzbDrone.Core/Localization/Core/en.json

```diff
@@ -482,7 +482,7 @@
   "Lists": "Lists",
   "ListSettings": "List Settings",
   "ListsSettingsSummary": "Import Lists, list exclusions",
-  "ListSyncLevelHelpText": "Movies in library will be removed or unmonitored if not in your list",
+  "ListSyncLevelHelpText": "Movies in library will be logged in Events, removed, or unmonitored based on your selection if not in any of your lists",
```

Comment:
Sounds a bit goofy

`unmonitored based on your selection if not in any of your lists`

Reply:
Suggestions welcomed 🤷‍♂️

Reply:
Rehashing all the options that are already in the list isn't needed IMO. Helptext should describe what the setting does or is for. `This action will be performed when movies are removed or drop off your list` or something.